workflows/readsimulator.nf

```diff
@@ -130,6 +131,25 @@ workflow READSIMULATOR {
         ch_simulated_reads  = ch_simulated_reads.mix(ch_metagenome_reads)
     }
 
+    //
+    // MODULE: Simulate wholegenomic reads
+    //
+    if ( params.wholegenome ) {
+        WGSIM (
+            ch_fasta.first(),
+            ch_input
+        )
+        ch_versions          = ch_versions.mix(WGSIM.out.versions.first())
+        ch_wholegenome_reads = WGSIM.out.fastq
+            .map {
+                meta, fastqs ->
+                    meta[0].outdir   = "wgsim"
+                    meta[0].datatype = "wholegenome"
+                    return [ meta[0], fastqs ]
```

Comment:
basically the same thing I commented on previous meta maps. But it is a style choice, so no worries :)